doc/manual/index.adoc

```diff
@@ -69,28 +71,77 @@ it before returning lacinia input data.
 
```

Comment:
Sorry, not in your PR, but the "What Sillsuit provides" section above doesn't seem very useful. What is "a bunch of stuff"?